src/main/kotlin/de/triplet/gradle/play/PlayPublishTask.kt

```diff
@@ -0,0 +1,46 @@
+package de.triplet.gradle.play
+
+import com.android.build.gradle.api.ApplicationVariant
+import com.google.api.client.googleapis.json.GoogleJsonResponseException
+import com.google.api.services.androidpublisher.AndroidPublisher
+import org.gradle.api.DefaultTask
+
+open class PlayPublishTask : DefaultTask() {
+
+    lateinit var extension: PlayPublisherPluginExtension
+
+    lateinit var variant: ApplicationVariant
+
+    lateinit var edits: AndroidPublisher.Edits
+
+    lateinit var editId: String
+
+    var playAccountConfig: PlayAccountConfig? = null
+
+    var service: AndroidPublisher? = null
+
+    fun publish() {
+        if (service == null) {
+            service = AndroidPublisherHelper.init(extension, playAccountConfig)
+        }
+
+        edits = service!!.edits()
```

Comment:
I don't like this. Unfortunately we can't get rid of this operator as `service` is a mutable property :(